setup.py

```diff
@@ -18,9 +19,10 @@
     keywords="microcosm",
     install_requires=[
         "click>=6.7",
-        "enum34>=1.1.6",
         "PyYAML>=3.12",
         "requests>=2.18.4",
+        # 09-21-2020: pin for enforce_white_space issue @pierce
+        "isort<5",
```

Comment:
@piercefreeman I've seen us pin this in our entrypoint.sh, do we actually want a version pin for a linter to be part of the `install_requires` of the package? cc/ @GabeLambda any input whether we should capture it here or elsewhere?

Reply:
This belongs more in the `lint` section of `entrypoint.sh`. This isn't a direct dependency of the project.